package-lock.json

```diff
@@ -2500,7 +2500,7 @@
     },
     "alphanum-sort": {
       "version": "1.0.2",
-      "resolved": "https://registry.npmjs.org/alphanum-sort/-/alphanum-sort-1.0.2.tgz",
+      "resolved": "https://registry.npm.taobao.org/alphanum-sort/download/alphanum-sort-1.0.2.tgz",
```

Comment:
Hey can you checkout this file from dev? Let's keep using npmjs.org for the registry

Reply:
Sorry, done, just saw this message